### PR TITLE
Square Pay Form

### LINKED ISSUE #198: Square pay form template

### PR DESCRIPTION
**Pull request by camwlsn**
Progress on square implementation, still have more work to do.

pay.js: Moved file location
.env: Front-end .env for keys
App.js: Link to Square form page if needed
SquareForm.js: Pay form template
next.config.js: Required to load the needed modules
pay.js: Server side. Generates token when a payment request is made.
SquarePay.js: Creates the payment form for customers to fill out and awaits for a token response from pay.js

close #198

### CODE REVIEW

File: MERN Scripts/mern-emsystem/src/.env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+SQUARE_ACCESS_TOKEN=EAAAELql3EigTRh3QbNoSnOuFLzYvuJNZw8AHqmUmk7v4M5Y3Gx9e0JDFUo22mQX
+SQUARE_APPLICATION_ID=sandbox-sq0idb-7JEDuestTK6Hj-fdj6C0FA
+SQUARE_LOCATION_ID=LS2QD8Y1ET9FY
```
Comment (amurph24) [inline MERN Scripts/mern-emsystem/src/.env:3]: I'm assuming that these will work on any machine that runs them? i.e. they are not unique to a particular ip or user

**Review by amurph24 — APPROVED** [review]
Looks good, as long as the environment variables work for anything

**Review by SpeiserK — APPROVED** [review]
Looks good, great start on square implementation 💯

**Review by zachpren0 — APPROVED** [review]
i like it

**Pull request merged**